sys-apps/portage/metadata.xml

```diff
@@ -11,6 +11,7 @@
   </maintainer>
   <use>
     <flag name="epydoc">Build html API documentation with epydoc.</flag>
+    <flag name="gentoo-dev">Set emerge --dynamic-deps=n by default.</flag>
```

Comment:
Maybe make it more generic, like 'enable features required for Gentoo ebuild development'. I think it'd be reasonable to look into adding other stuff here, like 'network-sandbox' or 'strict-keepdir'. Basically all the stuff that we want exploding in developer's faces rather than users'.

Reply:
Ok, updated.